tests/mocks/m4-xlarge.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
could you name this file to more clearly indicate what it's mocking? is this a response from IMDS?